.mvn/jvm.config

```diff
@@ -0,0 +1 @@
+-XX:-TieredCompilation -XX:TieredStopAtLevel=1 -XX:+UseParallelGC -Djava.awt.headless=true
```

Comment:
Older versions of Java and Maven don't seem to pick these up, but need to confirm with testing to make sure `-XX:+UseParallelGC` etc. don't break specific JVM versions.

Reply:
Yep, [JDK 8 seems fine](https://github.com/sgammon/guava/actions/runs/8224662966/job/22488758103).